packages/non-core/coffeescript-compiler/package.js

```diff
@@ -22,7 +22,7 @@ Npm.depends({
 });
 
 Package.onUse(function (api) {
-  api.use('babel-compiler@6.19.4');
+  api.use('babel-compiler@6.19.4||7.0.0');
```

Comment:
@GeoffreyBooth Good news: this syntax actually works!

Reply:
This is something, at least. It lets me as a package developer write something like `api.use('coffeescript@1||2');` (I assume that works, and not just exact `1.12.7||2.0.3`?)

It doesn’t really solve the issue of #9244, though, because to take advantage of this syntax you still need to fork a package. There still needs to be some way for end users to override the dependencies a package developer has set, if they haven’t thought to use this syntax or they picked versions that don’t work with other packages the user might want to install. But this is progress.

Reply:
Totally agree!